resources/web/map/js/trajectory_raw.js

```diff
@@ -5,7 +5,7 @@ const LNG_KM = 111.320, LAT_KM = 110.574;
 
 function decode(r){var n=r,t=[0,10,13,34,38,92],e=new Uint8Array(1.75*n.length|0),f=0,o=0,a=0;function i(r){o|=(r<<=1)>>>a,8<=(a+=7)&&(e[f++]=o,o=r<<7-(a-=8)&255)}for(var u=0;u<n.length;u++){var c,d=n.charCodeAt(u);127<d?(7!=(c=d>>>8&7)&&i(t[c]),i(127&d)):i(d)}r=new Uint8Array(e,0,f);var s=new TextDecoder().decode(r);while (s.slice(-1)=="\x00") s=s.slice(0,-1); return s;}
 
-mapboxgl.accessToken = decode("@ACCESS_TOKEN@");
+mapboxgl.accessToken = "pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw";
```

Comment:
oops... yea i agree (really tedious to remember to make sure its not committed)
https://swift-nav.atlassian.net/browse/CPP-919
just dropped a ticket here:

Reply:
@adrian-kong I'd really recommend a git hook here. It's not a long term solution because each team member would have to set up the hooks (they aren't automatically tracked as part of the git repo).

but if you create a file called commit-msg in your `path/to/swift-toolbox/.git/hooks/` folder that looks like this:
```
#!/bin/sh

# This catches matboxgl accessTokens
if git grep -E "mapboxgl.accessToken[ ]*=[ ]*\"[a-zA-Z0-9_.\-]{96}\""; then
    echo "Detected mapbox access token was comitted"
    exit 2
fi
```
this will not happen again.